src/current/v23.2/known-limitations.md

```diff
@@ -10,100 +10,61 @@ This page describes newly identified limitations in the CockroachDB {{page.relea
 
-    [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/75457)
+### PL/pgSQL Limitations
 
-- [`SELECT FOR UPDATE`]({% link {{ page.version.version }}/select-for-update.md %}) only places an unreplicated lock on the index being scanned by the query. This diverges from PostgreSQL, which aquires a lock on all indexes.
+#### Support for PL/pgSQL features
 
-    [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/57031)
+  - Nested blocks within PL/pgSQL blocks are not supported, and will lead to errors. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/114775)
+  - Cursors opened in PL/pgSQL do not execute their queries lazily, affecting performance and resource usage. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/111479)
+  - PL/pgSQL exception blocks cannot catch transaction retry errors. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/111446)
+  - PL/pgSQL does not support referring to arguments by ordinals (e.g., `$1`, `$2`), limiting flexibility in function and procedure definitions. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/114701)
+  - PL/pgSQL does not support `FOR` loop statements, limiting control flow capabilities in routines. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/105246)
+  - The `INTO` statement in PL/pgSQL does not support the `STRICT` option. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/107854)
 
-### Limitations for composite types
+#### Type Handling and Variable Declarations
 
-- {% include {{page.version.version}}/cdc/types-udt-composite-general.md %} The following limitations apply:
-    - {% include {{page.version.version}}/cdc/avro-udt-composite.md %}
-    - {% include {{page.version.version}}/cdc/csv-udt-composite.md %}
+- There are several known limitations regarding type handling and variable declarations in the current implementation of PL/pgSQL:
+  - PL/pgSQL does not support `RECORD` input arguments for UDFs and stored procedures. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/105713)
+  - Type coercion does not behave consistently with PostgreSQL, particularly when values are too wide for the target type. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/115385)
+  - PL/pgSQL lacks support for `RECORD`-type variables, which are dynamically typed in PostgreSQL. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/114874)
+  - Syntax for accessing members of composite types without parentheses is not supported. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/114687)
+  - `RAISE` statements cannot be annotated with names of schema objects related to the error (i.e., using `COLUMN`, `CONSTRAINT`, `DATATYPE`, `TABLE`, or `SCHEMA`). [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/106237)
```

Comment:
I thought of an additional RAISE statement limitation ([issue here](https://github.com/cockroachdb/cockroach/issues/117750)) - RAISE statements currently don't output to logs, only directly to the client.